.aws/src/datasyncLambda.ts

```diff
@@ -89,9 +90,13 @@ export class DatasyncLambda extends Resource {
    * @private
    */
   private createSqsForDlq() {
-    return new sqs.SqsQueue(this, 'datasync-target-lambda-dlq', {
+    const dlq = new sqs.SqsQueue(this, 'datasync-target-lambda-dlq', {
       name: `${config.prefix}-Datasync-Lambda-DLQ`,
     });
+
+    this.creatEventBridgeDlqAlarm(dlq);
```

Comment:
this says DLQ for event bridge, we expect the DLQ for eventBridge -> lambda integration to have close to 0 errors - so looks good! 

but what about sqs->lambda->DLQ errors?